docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -171,7 +171,7 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
     -DprojectArtifactId=getting-started-reactive \
     -DclassName="org.acme.quickstart.ReactiveGreetingResource" \
     -Dpath="/hello" \
-    -Dextensions="resteasy-mutiny, resteasy-jackson"
+    -Dextensions="resteasy-reactive, rest-client"
```

Comment:
Why the rest-client?

It is not needed AFAIK

Reply:
I need it for using `@@SseElementType` [here](https://github.com/aureamunoz/quarkus-quickstarts/blob/6ab7eb483f4f0931e6d609f461ea880df3c777d8/getting-started-reactive/src/main/java/org/acme/getting/started/ReactiveGreetingResource.java#L37)

Reply:
OK, that is wrong.

You should use `@org.jboss.resteasy.reactive.RestSseElementType` instead and remove the rest-client dependency

Reply:
OK. I didn't know it. Thank you

Reply:
org.jboss.resteasy.annotations.jaxrs.PathParam isn't in `quarkus-resteasy-reactive` dependency, is it expected?

Reply:
Yes, it is expected. You can omit the annotation entirely and the code should work properly